modules/data_release/ajax/AddPermission.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * Add permissions through Ajax, how crazy.
+ *
+ * PHP Version 5
+ *
+ *  @category Loris
+ *  @package  Data_Release
+ *  @author   Justin Kat <justinkat@gmail.com>
+ *  @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ *  @link     https://github.com/aces/Loris
+ */
+
+$DB =& Database::singleton();
+
+if ($_POST['action'] == 'addpermission') {
+    $userid          = $_POST['userid'];
+    $data_release_id = $_POST['data_release_id'];
+    $success         = $DB->insert(
+        'data_release_permissions',
+        array(
+         'userid'          => $userid,
+         'data_release_id' => $data_release_id,
+        )
+    );
+    header("Location: /data_release/?addpermissionSuccess=true");
```

Comment:
This is missing the baseURL so won't work on things not installed directly on the host @Jkat